x/programs/rust/examples/counter/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-use wasmlanche_sdk::{params, program::Program, public, state_keys, types::Address};
+use wasmlanche_sdk::{params, public, state_keys, types::Address, Program};
```

Comment:
Typically in Rust, it's only larger libraries (that have a bunch of feature flags) that force you to access commonly used types through modules. The most commonly used types should almost always be accessible from the root of a crate.